charts/vector/values.yaml

```diff
@@ -178,7 +178,19 @@ tolerations: []
 
 # affinity -- Allow Vector to schedule using affinity rules
 ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
-affinity: {}
+affinity:
+  ## Scale across different AZs by default.
+  podAntiAffinity:
+    preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 100
+        podAffinityTerm:
+          labelSelector:
+            matchExpressions:
+              - key: app.kubernetes.io/name
+                operator: In
+                values:
+                  - vector
```

Comment:
This would not be ideal depending on our story around installing Vector as both an Agent and Aggregator in the same cluster.